src/apps/explorer/components/OrderWidget/index.tsx

```diff
@@ -32,6 +32,7 @@ export const OrderWidget: React.FC = () => {
   return (
     <OrderDetails
       order={order}
+      orderId={orderId}
```

Comment:
I see why you are doing this, but I don't think it makes sense for 2 reasons:

1. `order` contains `orderId`
2. `orderId` input might not be an orderId

I would prefer to load this page only when we know it's an order, or at least we got a valid orderId (matches orderId regex)

Reply:
Ok, the problem with point 1, is that when you try to query the API (even with valid id) it returns null.

I filtered load this page how to you suggested in [Order](https://github.com/gnosis/gp-ui/blob/768-show-notfound-under-search/src/apps/explorer/pages/Order.tsx#L13) component.